mattlianje/gregarious-wolf
# Combined Amplitude and MFCC Feature extraction

- Amplitude and MFCC Analyses are now in one function

- Did not combine pitch as it uses a different package and would require a re-do of how we get pitch. Kept as separate (I know we're not even sure if using pitch or not anyways)

**HOW TO TEST**
The usual:
- cd `audio-analysis`
- make sure you have `secrets.py` in the directory
- turn on your _localhost_
- run `python main.py log`
- should work same old, slightly faster for longer videos
